licenses/cla-individual.md

```diff
@@ -222,3 +222,5 @@ Andreas Abeck, @andreasabeck 2015/03/18
 Hong-Yi Dai, @reflectionalist 2015/04/04
 
 heroboy, @heroboy 2015/04/19
+
+vouko, @vouko 2015/05/02
```

Comment:
> How would you know this is a real name ?

Whether he knows or not does not matter. He also does not know whether you stole the code you contributed or not. It is about having done all he could to prevent "fraud" and accepting pseudonyms is not good.